<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Projects</title>

<script src="site_libs/header-attrs-2.15/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Sarah Arcos</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="projects.html">Projects</a>
</li>
<li>
  <a href="r-fridays.html">R fridays</a>
</li>
<li>
  <a href="cv.html">CV</a>
</li>
<li>
  <a href="network.html">Network</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="contact.html">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/saraharcos">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/sarah_arcos">
    <span class="fa fa-twitter fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Projects</h1>

</div>


<p><br/></p>
<div id="projects-in-the-lauring-lab" class="section level2">
<h2>Projects in the Lauring Lab</h2>
<p><br/> <br/></p>
<div id="using-mi-networks-to-study-co-evolution-in-the-h3n2-polymerase"
class="section level3">
<h3>1. Using MI networks to study co-evolution in the H3N2
polymerase</h3>
<p><img src="polymerase-01.png" /></p>
<p>Changes in the speed, fidelity, and drug resistance of RNA virus
polymerases (including Influenza A) require multiple mutations. One
example is that an initial mutation may confer drug resistance, but
destabilize the protein- a second mutation is then required to maintain
overall fitness. Co-evolution is one way to identify residues that may
have a functional relationship. In this project, I have devised a method
to use weighted Mutual Information to first identify co-evolving residue
pairs. I then use a <a
href="https://saraharcos.github.io/network">network visualization</a> to
identify higher-order interactions. A benefit of using networks is the
ability to account for genetic hitchhiking with Influenza genes that are
under strong selection (like HA or NA).</p>
<p><img src="mi-01.jpg" style="width:50.0%" /></p>
<p><br/> <br/></p>
</div>
<div
id="measuring-changes-in-the-speed-and-fidelity-of-the-h3n2-polymerase"
class="section level3">
<h3>2. Measuring changes in the speed and fidelity of the H3N2
polymerase</h3>
<p><img src="tradeoff-01.png" style="width:50.0%" /></p>
<p>The evolution of RNA virus polymerases is subject to a trade-off
between speed and fidelity. As replicative speed increases, the
polymerase makes more errors. While faster replication alone is
obviously beneficial to the virus, the increased mutational load means
that the faster speed comes at a cost.</p>
<p>We also know that RNA virus mutation rates are near the theoretical
limit where the mutational burden becomes too high to support efficient
replication.</p>
<p>In the lab, I am working to determine the replicative speed and
mutation rate of H3N2 strains covering 50 years of evolution using our
viral competition assays and the GFP-based mutation rate assay developed
by <a href="https://pubmed.ncbi.nlm.nih.gov/28598328/">Pauly et
al</a>.</p>
<p><img src="gfp_assay.jpg" style="width:105.0%" /></p>
<p><br/> <br/></p>
<hr />
</div>
</div>
<div id="projects-in-the-ascano-lab" class="section level2">
<h2>Projects in the Ascano Lab</h2>
<p><br/> <br/></p>
<div id="vir-clasp" class="section level3">
<h3>1. VIR-CLASP</h3>
<p><img src="vir_clasp-01.png" /></p>
<p>VIR-CLASP (Viral Cross-Linking And Solid-phase Purification) is a
technique I co-developed to study host-pathogen interactions at the
earliest stages of viral infection. This technique answers the question
<strong>What are the host RNA-binding proteins that directly interact
with pre-replicated RNA virus genomes?</strong> The main trick of
VIR-CLASP is to use the nucleoside analog 4SU to label RNA virus
genomes, followed by infection into unlabeled host cells. This allows
for the specific identification of protein interactions with incoming
virion RNA, and not later stages of viral replication or host mRNAs.
Follow the links below to access all the data and code from our
proteomics analysis of Chikungunya and Influenza A virus.</p>
<p><a
href="https://github.com/saraharcos/CHIKV_VIRCLASP_Analysis">VIR-CLASP
for Chikungunya Virus</a></p>
<p><a
href="https://github.com/saraharcos/IAV_VIRCLASP_Analysis">VIR-CLASP for
Influenza A Virus</a></p>
<p><a
href="https://www.ebi.ac.uk/pride/archive/projects/PXD015863">Proteomics
Data</a></p>
<p><a
href="https://pubmed.ncbi.nlm.nih.gov/32380061/">Publication</a></p>
<p><a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8489533/">Protocol</a></p>
<p><br/> <br/></p>
</div>
<div id="exploring-m6a-and-ythdf-clip-datasets" class="section level3">
<h3>2. Exploring m6A and YTHDF CLIP datasets</h3>
<p>YTHDF proteins regulate RNA metabolism through recognition of the RNA
modification N6-methyladenosine (m6A). While <em>in vitro</em>
biochemical data indicate that YTHDF proteins have a modest preference
for methylated RNA over unmethylated RNA, <em>in vivo</em> results
indicate widespread binding of YTHDF proteins to unmethylated mRNAs and
viral RNAs. This project aims to investigate the binding determinants
and biological implications of YTHDF recognition of unmethylated
RNAs.</p>
<p><a href="https://github.com/saraharcos/m6A_Exploratory">Exploring m6A
and YTHDF CLIP datasets</a></p>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
